.changeset/long-carrots-sleep.md

```diff
@@ -0,0 +1,15 @@
+---
+"@spectrum-css/search": major
+---
+
+## S2 Minimized search field
+
+The search component allows for a minimized state where the search field is collapsed to a button.
+
+### Anatomy
+
+The collapsed state consists of a single action button that only has a hover and disabled state. This state is triggered by the `is-collapsed` class on the search component. When the search field is in this state, the textfield is hidden and the search button is displayed. The button can be hovered and focused, and will expand the search field when clicked.
```

Comment:
since we added keyboard focus back in, can we update this first sentence just a touch so that it doesn't say that it "only" has hover and disabled states? You mention focus later in the paragraph, even if you decide not to change it. 👍